### PR TITLE
Add a method that exposes logging in TheoreticalSensitivity.

### PR DESCRIPTION
**Pull request by nh13**
@yfarjoun the motivation comes from wanting to call this method many times and not have so many logging messages.

Also, @davidbenjamin, could you possibly add a comment for the class explaining the motivation and reasoning behind this method?  Or provide a link to a paper/reference that does?  It would be great for folks who want more information about the motivation, math, and what it is trying to estimate.

### CODE REVIEW

File: src/java/picard/analysis/TheoreticalSensitivity.java [hunk 1]
```diff
@@ -156,7 +170,7 @@ public int draw() {
         }
 
         //get samples of sums of 0, 1, 2,. . .  N - 1 draws
-        public List<ArrayList<Integer>> sampleCumulativeSums(final int maxNumberOfSummands, final int sampleSize) {
+        public List<ArrayList<Integer>> sampleCumulativeSums(final int maxNumberOfSummands, final int sampleSize, final boolean withLogging) {
```
Comment (yfarjoun) [inline src/java/picard/analysis/TheoreticalSensitivity.java]: need to change the test as well...as it is currently failing.

**Comment by yfarjoun** [comment]
except for the failing test, this looks fine. I'm looking into finding a good home for the documentation (and to have it referenced in the code), in the meantime it can found here www.broadinstitute.org/~farjoun/white_papers/theoretical-het-sensitivity.pdf

**Comment by nh13** [comment]
@yfarjoun all set.

**Comment by yfarjoun** [comment]
thanks! 

:+1:

**Pull request merged**